src/rp2_common/hardware_pio/pio.c

```diff
@@ -240,18 +240,27 @@ void pio_sm_set_pins64(PIO pio, uint sm, uint64_t pins) {
     pio->sm[sm].execctrl = execctrl_saved;
 }
 
-void pio_sm_set_pins_with_mask(PIO pio, uint sm, uint32_t pinvals, uint32_t pin_mask) {
-    pio_sm_set_pins_with_mask64(pio, sm, pinvals, pin_mask);
+#ifndef pio_sm_set_pins_internal
```

Comment:
I'm sure this must be a stupid question, but why is line 243 doing
```
#ifndef pio_sm_set_pins_internal
```
when line 221 says
```
void pio_sm_set_pins_internal(PIO pio, uint sm, uint32_t pins) {
```
?